src/librustc_back/target/x86_64_pc_windows_msvc.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use target::Target;
+
+pub fn target() -> Target {
+    let mut base = super::windows_msvc_base::opts();
+    base.cpu = "x86-64".to_string();
+
+    Target {
+        // FIXME: Test this. Copied from linux (#2398)
```

Comment:
It appears that this is just being copied around, I copied this file from `x86_64-pc-windows-gnu` which has the exact same comment, so I'm not really sure what the implications are unfortunately :(. 

I'll add some words here to this effect though.